MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -72,6 +76,23 @@ public sealed class CreditCard
             Was3DSecureSuccessful = was3DSecureSuccessful;
         }
 
+        /// <summary>
+        /// [Obsolete("Legacy constructor for backwards compatibility")]
```

Comment:
Sorry, one last thing. Is there a reason we aren't applying this attribute to the constructor and only have it in the documentation? Applying it will cause an users to get a warning at compile time, discouraging its use.

Reply:
I just cargo culted it from the other obsolete constructor 😬

Reply:
Huh, I have no idea why that one is like that either.